Exec/hydro_tests/riemann_2d/inputs_c12

```diff
@@ -0,0 +1,106 @@
+#------------------INPUTS TO MAIN PROGRAM---------------
+#max_step = 1000                           #max of time steps
+stop_time = 0.25                            #stop_time
+
+#PROBLEM SIZE & GEOMETRY
+geometry.is_periodic = 1    1    1         # 1 if its periodic and 0 otherwise
+geometry.coord_sys   = 0                   # 0 is cartesian
+geometry.prob_lo     = 0    0    0         # lowest boundary on each domain
+geometry.prob_hi     = 1    1    1         # highest boundary on each domain
+amr.n_cell           = 400 400 400         # number of cells
+
+# >>>>>>>>>>>> BC FLAGS <<<<<<<<<<<<<<<<
+# 0 = Interior          3 = Symmetry
+# 1 = Inflow            4 = SlipWall
+# 2 = Outflow           5 = NoSlipWall
+# >>>>>>>>>>>> BC FLAGS <<<<<<<<<<<<<<<<
+castro.lo_bc         = 0  0  0
+castro.hi_bc         = 0  0  0
+
+#WHICH PHYSICS
+castro.do_hydro = 1
+castro.do_react = 0
+
+#TIME STEP CONTROL
+
+castro.cfl           = 0.6
+castro.init_shrink   = 1.0
+castro.changemax     = 1.1
+#castro.fixed_dt      = 
+
+
+#DIAGNOSTIC & VERBOSITY
+castro.sum_interval = 1
+castro.v            = 1
+amr.v               = 1
+
+
+#REFINEMENT / REGRIDDING
+amr.max_level       = 0             # maximum level number allowed
+amr.ref_ratio       = 2 2 2 2       # refinement ratio
+amr.regrid_int      = 2             # how often to regrid
+amr.blocking_factor = 8             # block factor in grid generation
+amr.max_grid_size   = 256           # maximum default value
+
+amr.refinement_indicators = presserr pressgrad
+
+amr.refine.presserr.value_greater = 2
+amr.refine.presserr.field_name = pressure
+amr.refine.presserr.max_level = 3 #Since blocking_factor = 2**3, 3 is the max level chosen for refinement.
```

Comment:
blocking_factor means that the grid sizes will be multiples of blocking_factor -- it doesn't affect the maximum level of refinement.  That is controlled by max_level

Reply:
I see, so the blocking factor essentially constraint the size of the cells that are chosen for the refinement.